xray/audit/java/mvn.go

```diff
@@ -35,10 +38,23 @@ func runMvn(buildConfiguration *utils.BuildConfiguration, insecureTls, ignoreCon
 			log.Debug("Using resolver config from " + configFilePath)
 		}
 	}
+	if useWrapper {
+		useWrapper, err = isMvnWrapperExist()
+		if err != nil {
+			return
+		}
+	}
 	// Read config
-	vConfig, err := utils.ReadMavenConfig(configFilePath)
+	vConfig, err := utils.ReadMavenConfig(configFilePath, useWrapper)
 	if err != nil {
 		return err
 	}
-	return mvnutils.RunMvn(vConfig, "", buildConfiguration, goals, 0, insecureTls, true)
+	return mvnutils.RunMvn(vConfig, "", buildConfiguration, goals, 0, insecureTls, useWrapper, true)
+}
+
+// This function assumes that the Maven wrapper is in the root directory.
+// The --project-dir option of Maven won't work in this case.
```

Comment:
This comment line seem irelevant for this function.

Reply:
Done